RELEASE.md

```diff
@@ -13,7 +13,13 @@ The Kudo Project is released on an as-needed basis. The process is as follows:
 
 ## Release Process
 
-The official binaries for Kudo are created using [goreleaser](https://goreleaser.com/) for the release process through the circleci release job. The [.goreleaser.yml](.goreleaser.yml) defines the binaries which are supported for each release.
+The official binaries for Kudo are created using [goreleaser](https://goreleaser.com/) for the release process through the circleci release job. The [.goreleaser.yml](.goreleaser.yml) defines the binaries which are supported for each release.  The circle-ci release process has not been provided with credentials for Github and Docker hub yet.  The current process to initiate a release is:
+
+1. Ensure you have credential `GITHUB_TOKEN` set.
```

Comment:
Do we still need this @alenkacz ?

Reply:
I believe this is needed to be able to write releases to the github release page so probably yes.